app/scripts/firebase/main.js

```diff
@@ -9,26 +9,58 @@ function ArtHop() {
   this.logoutButton = document.getElementById('logout');
   this.nameDisplay = document.getElementById('sample-text');
   this.venues = [];
+  this.users = [];
 
   //Button listeners
   this.loginButton.addEventListener('click', this.signIn.bind(this));
   this.logoutButton.addEventListener('click', this.signOut.bind(this));
 
   this.initFirebase();
   this.loadVenues();
+  this.loadUsers();
 }
 
 ArtHop.prototype.loadVenues = function() {
   this.venuesRef = this.database.ref('venues');
   this.venuesRef.off();
   var self = this;
-  var venues = [];
 
   this.venuesRef.once('value').then(function(data) {
     self.venues = data.val(); 
   });
 }
 
+ArtHop.prototype.loadUsers = function() {
+  this.usersRef = this.database.ref('users');
+  this.usersRef.off();
+  var self = this;
+
+  function appendUsers(data) {
+    var users = data.val();
+    for(var u in users) {
```

Comment:
I feel like this could be done some other way, but nothing comes to mind... So just leaving a message in case something gets thought up. Probably something with map->reduce? or using some model collection. Either way, it works.